docs/contributing.md

```diff
@@ -16,7 +16,7 @@ directly
 Otherwise, create an issue in
 [dasharo-issues repisotory](https://github.com/Dasharo/dasharo-issues/issues/new/choose).
 
-## Guidelines
+# Guidelines
```

Comment:
Is it supposed to be a part of this PR?

Reply:
I would propose to skip this, and also squash the commits adding tests, with a meaningful commit msg.

Reply:
Change dropped.